Classes/common/HTTP/CDTURLSessionTask.m

```diff
@@ -37,6 +43,10 @@ @interface CDTURLSessionTask ()
  */
 @property (nonatomic, strong) NSArray *requestInterceptors;
 
+@property (nonnull, nonatomic, strong) NSArray *responseInterceptors;
+
+@property int remainingRetries;
```

Comment:
hmm I want to say no, but I can't be sure without running ReplicationAcceptance with it set to nonatomic